app/controllers/api/v1/organizations_controller.rb

```diff
@@ -16,6 +16,11 @@ class Api::V1::OrganizationsController < Api::ApiController
                     :title,
                     :introduction].freeze
 
+  CONTENT_FIELDS = [:description,
```

Comment:
should these be moved to the shared operations module if they aren't used in the controller?